c++/src/capnp/fuzz-run.c++

```diff
@@ -0,0 +1,21 @@
+#include "test-util.h"
```

Comment:
Can we add some comments to this file explaining what it's for and how to use it?

Reply:
Adding something